testing/src/cypress/e2e/04-api-institution/03-instition-manage-designation.cy.ts

```diff
@@ -0,0 +1,115 @@
+import InstitutionHelperActions from "../../custom-command/institution/common-helper-functions.cy";
+import Authorization, {
+  ClientId,
+} from "../../custom-command/common/authorization";
+import CommonRegex from "../../custom-command/common/commonRegex";
+
+const institutionHelperActions = new InstitutionHelperActions();
+const commonRegexObject = new CommonRegex();
+
+const USERNAME = institutionHelperActions.getUserNameForApiTest();
+const PASSWORD = institutionHelperActions.getUserPasswordForApiTest();
+const TOKEN_URL = institutionHelperActions.getApiUrlForKeyCloakToken();
+
+describe("[Designation details] ", () => {
+  let token: string;
+
+  before(async () => {
+    const authorizer = new Authorization();
+    token = await authorizer.getAuthToken(
+      USERNAME,
+      PASSWORD,
+      ClientId.Institution,
+      TOKEN_URL
+    );
+  });
+
+  it("Verify GET call for designation-agreement endpoint", () => {
+    cy.request({
+      method: "GET",
+      url: institutionHelperActions.getApiForDesignationAgreement(),
+      followRedirect: false,
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    }).then((response) => {
+      const body = response.body;
+      expect(response.status).to.be.eq(200);
+      body.forEach((designation: JSON) => {
+        expect(designation.designationId).to.be.a("number");
+        expect(designation.designationStatus).to.be.oneOf([
+          "Approved",
```

Comment:
mm, i was thinking can have a enum for statuses

Reply:
We can install some package like this one: https://www.npmjs.com/package/http-status-codes

Reply:
Will look at these approaches